src/vendor/Export2Excel.js

```diff
@@ -1,6 +1,7 @@
 /* eslint-disable */
 require('script-loader!file-saver');
 import XLSX from 'xlsx'
+import language from '@/lang'
```

Comment:
Please not change this native class from PanJianChen, revert it and implement a boolean conversion to string before send to export